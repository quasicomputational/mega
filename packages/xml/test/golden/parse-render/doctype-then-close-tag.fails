<!DOCTYPE hello>
</hello>
<hello>foo</hello>
