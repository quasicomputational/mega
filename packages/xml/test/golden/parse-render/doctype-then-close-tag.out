Error: Close tag follows doctype at (2, 1).
